packages/atmosphere/src/providers/twitter/processor.ts

```
@@ -979,14 +983,15 @@ export const buildAPITwitterStatus = async (
   }
 
   /* If a language is specified in API or by user, let's try translating it! */
+  const normalizedTarget =
+    typeof language === 'string' ? normalizeLanguage(language) : '';
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)